test/dummy/db/schema.rb

```diff
@@ -48,14 +48,15 @@
     t.integer "tick_count", default: 0, null: false
     t.integer "tick_total"
     t.string "job_id"
-    t.bigint "cursor"
+    t.integer "cursor"
```

Comment:
Yeah sqlite can't dump it back into a bigint, but I think we should keep it as a bigint since it's what's in the original migration.

Reply:
Yeah my bad, didn't even see this! Thanks for flagging 👍